src/libs/intents/printBase64Doc.ts

```diff
@@ -0,0 +1,82 @@
+import { format } from 'date-fns'
+import RNFS from 'react-native-fs'
+import RNPrint from 'react-native-print'
+import Toast from 'react-native-toast-message'
+
+import MiniLog from 'cozy-minilog'
+
+import { getErrorMessage } from '/libs/functions/getErrorMessage'
+import { t } from '/locales/i18n'
+
+const log = MiniLog('intents:print')
+
+// File Handling
+const getPrintFolderPath = (): string => {
+  return `${RNFS.TemporaryDirectoryPath}/print`
+}
+
+const getTemporaryPrintFilePath = (fileExtension: string): string => {
+  const date = format(new Date(), 'yyyyMMddHHmmssSSS')
+  const tempFileName = `temp_print_${date}.${fileExtension}`
+  const path = `${getPrintFolderPath()}/${tempFileName}`
+  return path
+}
+
+const ensurePrintFolder = async (): Promise<void> => {
+  await RNFS.mkdir(getPrintFolderPath())
+}
+
+const safeDeleteFile = async (path: string): Promise<void> => {
+  try {
+    if (await RNFS.exists(path)) {
+      await RNFS.unlink(path)
+    }
+  } catch (err) {
+    log.error(`Error deleting document: ${getErrorMessage(err)}`)
+  }
+}
+
+const writeFile = async (filePath: string, content: string): Promise<void> => {
+  return await RNFS.writeFile(`${filePath}`, content, 'base64')
```

Comment:
nit: we don't need this templated string anymore here.
```suggestion
  return await RNFS.writeFile(filePath, content, 'base64')
```

Reply:
✅